acora/_acora.py

```diff
@@ -42,8 +42,8 @@ class _MachineState(object):
     """Trie node state for the automaton.
     """
     def __init__(self):
-        self.id = None
-        self.letter = None
+        self.id = 0
+        self.letter = 0
```

Comment:
Is this change related?

Reply:
yes. Otherwise Cython complains that unsigned long cannot be assigned None